doc/user/guide.rst

```diff
@@ -237,10 +237,10 @@ For a cleverer approach that we will someday implement, read `the
 Beachcomber's Dilemma
 <http://ivory.idyll.org/blog/beachcombers-dilemma.html>`__.
 
-Validating and comparing assemblies
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. Validating and comparing assemblies
+.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-More here soon :).
+.. More here soon :).
```

Comment:
ohh, that's what that does...

I was thinking that's a toctree type thing.

On Tue, Aug 11, 2015 at 4:21 PM Michael R. Crusoe <notifications@github.com>
wrote:

> In doc/user/guide.rst
> <https://github.com/dib-lab/khmer/pull/1222#discussion_r36811715>:
>
> >
> > -More here soon :).
> > +.. More here soon :).
>
> Which is why I commented it out :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1222/files#r36811715>.
>